cardboard/settings.py

```diff
@@ -311,6 +311,10 @@ class TaskPriority(Enum):
         },
     },
     "loggers": {
+        "": {
+            "handlers": ["console"],
+            "level": os.getenv("DJANGO_LOG_LEVEL", "INFO"),
```

Comment:
I think this env variable is supposed to be specific to the logging messages that django itself generates so I don't think we should reuse it here as this is for the logging messages we created ourselves. IMO better to either hardcode it, condition it on the DEBUG setting, or create a separate env variable

Reply:
Sounds good. I just changed the variable to `DJANGO_APP_LOG_LEVEL` for flexibility in the future.